policyengine_us/variables/household/income/person/weekly_hours_worked.py

```diff
@@ -39,15 +39,21 @@ def formula(person, period, parameters):
         original_emp = person("employment_income_before_lsr", period)
         original_self_emp = person("self_employment_income_before_lsr", period)
         original_earnings = original_emp + original_self_emp
-        lsr_relative_change = np.where(
-            original_earnings == 0, 0, income_effect / original_earnings
+
+        lsr_relative_change = np.divide(
+            income_effect,
+            original_earnings,
+            out=np.zeros_like(
```

Comment:
To avoid awkward line breaks.
```suggestion
            # Assign no LSR change to people with no original earnings to avoid dividing by zero.
            out=np.zeros_like(income_effect, dtype=np.float32),
            where=original_earnings != 0,
```

Reply:
Try `make format` and please add the comment I suggested